package.json

```diff
@@ -42,7 +42,8 @@
   "dependencies": {
     "@bugsnag/js": "^8.2.0",
     "@modelcontextprotocol/sdk": "^1.15.0",
-    "node-cache": "^5.1.2"
+    "node-cache": "^5.1.2",
+    "zod": "^3"
```

Comment:
I left the version loose so we use the same version as the current modelcontextprotocol/sdk package